mzLib/Test/Transcriptomics/TestDecoyGenerator.cs

```diff
@@ -0,0 +1,265 @@
+﻿using NUnit.Framework;
+using Omics.Modifications;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework.Interfaces;
+using Transcriptomics;
+using Transcriptomics.Digestion;
+using UsefulProteomicsDatabases.Transcriptomics;
+using UsefulProteomicsDatabases;
+
+namespace Test.Transcriptomics
+{
+    [TestFixture]
+    [ExcludeFromCodeCoverage]
+    internal class TestDecoyGeneration
+    {
+        public static string ModomicsUnmodifiedFastaPath => TestDbLoader.ModomicsUnmodifedFastaPath;
+
+        [Test]
+        public static void TestReverseDecoy_Simple()
+        {
+            var oligos = new List<RNA>()
+            {
+                new RNA("GUUCUG"),
+                new RNA("GUGCUA"),
+            };
+            var decoys = RnaDecoyGenerator.GenerateDecoys(oligos, DecoyType.Reverse, 1);
```

Comment:
we should discuss slided decoys. maybe that isn't a good option for rna or dna.

Reply:
agreed. It should be considered as there may be issues with specific ion types in a reversed sequence

Reply:
should there be a comment on this in RnaDecoyGenerator.GenerateDecoys ?

Reply:
added comment to the TODO list in RnaDecoyGenerator